.github/workflows/nightly-cleanup.yml

```diff
@@ -52,13 +52,23 @@ jobs:
       - name: Find CI clusters
         run: |
           . venv/bin/activate
-          CI_CLUSTERS=$(openstack server list | grep --only-matching 'slurmci-RL.-[0-9]\+'  | sort | uniq)
-          echo "ci_clusters=${CI_CLUSTERS}" >> GITHUB_ENV
+          CI_CLUSTERS=$(openstack server list | grep --only-matching 'slurmci-RL.-[0-9]\+'  | sort | uniq || true)
+          if [[ -z "$CI_CLUSTERS" ]]; then
+            echo "No matching CI clusters found."
+          else
+            echo "Found clusters: $CI_CLUSTERS"
+            echo "ci_clusters=${CI_CLUSTERS}" >> GITHUB_ENV
+          fi
         shell: bash
       
       - name: Delete clusters if control node not tagged with keep
         run: |
           . venv/bin/activate
+          if [[ -z "${CI_CLUSTERS}" ]]; then
```

Comment:
this was set to `ci_clusters` in previous step, I think `CI_CLUSTERS` will be empty in this step